crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -147,6 +147,37 @@ impl ZombienetCommand {
 				}
 
 				spinner.stop(result);
+
+				// Check for any specified channels
+				if zombienet.hrmp_channels() {
+					let relay_chain = zombienet.relay_chain();
+					match RelayChain::from(relay_chain) {
+						None => {
+							log::error(format!("🚫 Using `{relay_chain}` with HRMP channels is currently unsupported. Please use `paseo-local` or `rococo-local`."))?;
+						},
+						Some(_) => {
+							use tokio::time::sleep;
+							let spinner = cliclack::spinner();
```

Comment:
```suggestion
							let spinner = spinner();
```

Any reason not to do a top level import of spinner?

Reply:
Name clash with import. Have changed to `progress` in 2e71e04b